datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/config.py

```diff
@@ -106,7 +106,7 @@ def config(ctx, check, sync, verbose):
                             files_failed[example_file_path] = True
                             check_display_queue.append(
                                 lambda example_file=example_file, **kwargs: echo_failure(
-                                    f'File `{example_file}` needs to be synced', **kwargs
+                                    f'File `{example_file}` is not in sync, run "ddev validate config -s"', **kwargs
```

Comment:
Instead of more verbose message for every invalid config, maybe we can specify the sync message as a warning at the end of the  output.

Adding something like the following after L129

```
echo_warning(f'Sync configuration files by running "ddev validate config --sync" and save the changes.')
```

With resulting output that is more readable:
<img width="724" alt="Screen Shot 2020-09-25 at 4 21 24 PM" src="https://user-images.githubusercontent.com/15065007/94312597-2f55be80-ff4b-11ea-949a-3362253522ad.png">